### PR TITLE
evse.cpp: fix modbus overload when loadbalance disabled

### PR DESCRIPTION
**Pull request by dingo35**
Fixes bug, so this PR should be applied before releasing 1.4.0.

### CODE REVIEW

**Review by koen-serneels — CHANGES_REQUESTED** [review]

File: SmartEVSE-3/src/evse.cpp [hunk 1]
```diff
@@ -1844,13 +1843,15 @@ uint8_t PollEVNode = NR_EVSES;
...
                     }
+                    ModbusRequest = 20;
                 case 20:                                                         // EV kWh meter, Current measurement
                     // Request Current if EV meter is configured
                     if (EVMeter) {
 #ifdef LOG_INFO_MODBUS
                         _Serialprintf("ModbusRequest %u: Request EVMeter Current Measurement\n", ModbusRequest);
 #endif
                         requestCurrentMeasurement(EVMeter, EVMeterAddress);
+                        ModbusRequest++;
```
[the remark this answers is not in the document]
Comment (dingo35) [reply SmartEVSE-3/src/evse.cpp:1854]: You are absolutely right, a new case would be skipped later; as was the case already with the old code :-( . But still it is needed because of the ugly ModbusRequest++ in the switch statement. 
I'll explain in detail, hope you don't find me pedantic :-):
The switch (ModbusRequest++) works like this: if the routine is entered with ModbusRequest (I'll shorten this to MBR)  MBR = 1, the switch matches with case 1: , and then IMMEDIATELY increases MBR by 1. So all statements in case 1 have MBR = 2 (as you can see, all the printed "ModbusRequest %u" messages have one increased to the case statement).

I thought about reworking this ugly construct but was afraid to touch it, afraid of unknown side-effects. So I decided to keep the ugly switch (ModbusRequest++) and and work with it.
In line 1846 I have to add the MBR = 20 statement, since it can be entered from case 13 - 19, so a MBR++ wouldnt  work.
The side effect of this is that now case 20: can be entered in two ways: 
1) if MBR=20 and the routine is entered; case 20 is matched , MBR immediately increased to 21, and if EVMeter is installed immediately increased to 22. Case 21 would be missed but luckily there is no such case. Also, turns out, because MBR is increased every time, there is no situation where the while loop is entered with MBR=20; because there is no situation where the while loop is left with MBR=20. 
2) if MBR between 13 and 19 and Loadbl is disabled, it will leave case 19 with MBR=20 and match case 20. It will execute case 20 code with MBR=20, if EVMeter is installed it will increase to MBR=21 and if not installed it will also increase to MBR=21. So next time routine is entered  it would match case 21 (although there is none now, but I can tell you my energy_api PR has one, and it works :-) ), just like intended.

I hope I have explained it enough; I hate this kind of ugly programming, if you want I can look howto get rid of the ugly  switch (ModbusRequest++) construct, but we would deviate a lot from Stegen and it would be entering uncharted territories with unknown side effects; if you want to go that way just let me know, although I think we should not want this in v1.4.0 ....

**Comment by dingo35** [comment]
Ok I found a much more elegant solution, please re-review!
